src/plot_api/plot_api.js

```diff
@@ -2092,9 +2092,6 @@ function _relayout(gd, aobj) {
     var oldWidth = gd._fullLayout.width,
         oldHeight = gd._fullLayout.height;
 
-    // coerce the updated layout
-    Plots.supplyDefaults(gd);
```

Comment:
fixed that bad-reference bug in https://github.com/plotly/plotly.js/pull/1403/commits/9f89dd690a638a4979d77d050667a1557baf91e5 - it was unrelated to removing this `supplyDefaults`, which I still believe is OK to do.